src/ast/InterfaceDecl.ts

```diff
@@ -0,0 +1,12 @@
+import {Content} from "./Content";
+
+export class InterfaceDecl extends Content {
```

Comment:
Nitpick

Can we add some comments to classes describing which part of the EBNF it refers to? It would save us a bit of switching back and forth between code and the EBNF when we develop.

Reply:
Agreed, updated